client/components/index.js

```diff
@@ -6,5 +6,6 @@
 export {default as Main} from './main'
 export {default as UserHome} from './user-home'
 export {Login, Signup} from './AuthForm'
+export {AllWatchesContainer as AllWatches} from './AllWatches'
```

Comment:
change to export {default as AllWatches} from './AllWatches. See previous comment.